cmd/cli/query/query.go

```diff
@@ -0,0 +1,20 @@
+package query
+
+import (
+	"fmt"
+	"github.com/go-resty/resty/v2"
+	"github.com/nextdotid/proof-server/controller"
+	"strings"
+)
+
+func QueryProof(platform string, identity string) {
+	req := controller.ProofQueryRequest{}
+	req.Platform = platform
+	req.Identity = strings.Split(req.Identity[0], ",")
+
+	client := resty.New()
+	url := fmt.Sprintf("http://localhost:9800/v1/proof?identity=%s&platform=%s", identity, platform)
+	resp, _ := client.R().EnableTrace().Get(url)
+
+	fmt.Println(resp)
+}
```

Comment:
Don't put this module into `/cmd/`, since it's a special directory for `go`.

Move this `cli` package out of `/cmd/`.